packages/docs/src/pages/writing-css.mdx

```diff
@@ -66,14 +68,14 @@ In the example below the two selectors (textarea and input) both alter the color
 
 ```jsx
 <div
-  css={{
+  css={css({
     'textarea,input': {
```

Comment:
nit: `textarea,input` could be changed to `textarea, input` (with a space after the comma) for better readability maybe 🤔